raiden/tests/unit/test_transport.py

```diff
@@ -11,11 +13,25 @@
 slogging.configure(':DEBUG')
 
 
+class IntervalCheck(object):
```

Comment:
Maybe write a docstring here, saying that at class initialization and at every function invocation the Interval timer is restarted? And maye `IntervalTimer` is a more fitting name?